elytron/src/test/java/org/wildfly/extension/elytron/TlsTestCase.java

```diff
@@ -471,6 +471,19 @@ public void testRevocationListsDp() throws Throwable {
         Assert.assertTrue(services.executeOperation(operation).get(OUTCOME).asString().equals(FAILED)); // not realoadable
     }
 
+
+    /**
+     * Verifies accepted issuers are sent when CRLs are configured.
+     * Verifies ELY-2057 (No acceptedIssuers sent when CRLs are configured) is resolved.
+     */
+    @Test
+    public void testAcceptedIssuersWithCRLs() throws Throwable {
+        ServiceName serviceName = Capabilities.TRUST_MANAGER_RUNTIME_CAPABILITY.getCapabilityServiceName("trust-with-crl");
+        TrustManager trustManager = (TrustManager) services.getContainer().getService(serviceName).getValue();
+        Assert.assertNotNull(trustManager);
+        Assert.assertTrue(((SSLDefinitions.ReloadableX509ExtendedTrustManager) trustManager).getAcceptedIssuers().length > 0);
```

Comment:
Instead of casting to `SSLDefinitions.ReloadableX509ExtendedTrustManager`, you should be able to just cast to `X509TrustManager` instead. Then, `ReloadableX509ExtendedTrustManager` can be made `private` again.

Reply:
Thanks! I updated it